content/special-topics/color-management/color-spaces.md

```diff
@@ -6,21 +6,27 @@ draft: false
 author: "people"
 ---
 
-darktable's input images are either RGB files (like JPEGs or TIFFs) or camera raws. Both represent colors by a combination of red, green and blue. Most of the actual image processing takes place in a large RGB "working profile" space, with some (mostly older) modules internally working in the CIELAB color space (often just called “Lab”). The final output of the image processing pipeline is once again in an RGB space shaped for either monitor display or the output file.
+darktable's input images are either RGB files (like JPEGs or TIFFs) or camera raws. Both store visual information as a combination of primary colours like red, green and blue which, together, describe a light emission to be later recreated by a display.
+
+It is important to mention that colour only exists in the human brain, as a product of the decomposition of a light emission by the cone cells in the retina. What RGB files do is describing light emissions with a similar method, for technical reasons. However, the definition of each primary colour depends on the properties of each RGB space and might look very different to human eyes from what humans call "red", "green" or "blue".
+
+Notice of warning, RGB is not really a color, but rather a light emission being encoded for computational purposes, that will later be physically created by your display, and interpreted as colour only once it enters your brain through your retina. Calling RGB "colour" is a shortcut.
```

Comment:
I'm not sure about the sentences "RGB is not really a color" and "Calling RGB "color" is a shortcut". Agree with the sentiment just not sure about the term RGB to represent a single "color". We're using RGB to refer to the color "space". Perhaps something like "A single RGB value is not really a color" but it feels a bit clumsy.

Aside: we're using American spellings to be consistent with the application, so "color" rather than "colour".

Reply:
Well, an RGB vector can encode a colo~~u~~r, if you abide by the web definition of color. For example, RGB = { 0.75, 0.75, 0.5 } "is" orange and you will find many online converters that will do it for you. The original text mentioned "RGB colors". 

What I'm trying to fight here is the idea that any RGB vector is **directly** a color, because if people stick to that, many things in the scene-referred workflow will make zero sense. It's much easier to conceptualize RGB as lights you dim and mix to reconstruct the original spectrum. That's also something you can experiment with IRL with dimable light bulbs and gels. 

Point being, we see colors, but we work on light, and the light -> color conversion is very different depending on the sensor.